### ISSUE TITLE
Popup option missing from default usage and panning problem

### ISSUE THREAD

**Issue by Toutouwai** [marioestrada/jQuery-gMap#4]
The "popup" option for having a marker's info window open by default is only available on the addMarker additional method. It would be nice to have it available via the default usage. Either that or add "address" as an option on the addMarker method.

Also, when popup is set to true, it doesn't pan the map as it should - it seems to not pan down far enough meaning that the top of the info window can be cut off. Compare the panning that occurs when an info window is opened by click versus open using popup: true.

**Comment by marioestrada** [comment]
Thanks, I'll look into it when I have the time.

**Comment by wjbryant** [comment]
The missing popup option was confusing to me as well. Adding `marker.popup` to the array of arguments on lines 196 and 202 should fix this.

Also, changing line 168

```
popup && infowindow.open($gmap, gmarker);
```

to

```
if (popup) {
    google.maps.event.addListenerOnce($gmap, 'tilesloaded', function () {
        infowindow.open($gmap, gmarker);
    });
}
```

seems to help with the info window auto-panning problem.

**Comment by marioestrada** [comment]
Please try this update and see if this fixes the issue:
https://raw.github.com/marioestrada/jQuery-gMap/e53cda5f28182e91eb03ed4aaa61aff556a94f70/jquery.gmap.js

**Comment by wjbryant** [comment]
That fixed it for me. Thanks!

**Comment by marioestrada** [comment]
Fixed in 2.1.2
